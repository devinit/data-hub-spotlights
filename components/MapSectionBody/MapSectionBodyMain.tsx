import React, { FunctionComponent } from 'react';

const MapSectionBodyMain: FunctionComponent = ({ children }) => {
  return (
    <div className="spotlight__main spotlight__interactive">
      { children }
    </div>
  );
};

export { MapSectionBodyMain };
