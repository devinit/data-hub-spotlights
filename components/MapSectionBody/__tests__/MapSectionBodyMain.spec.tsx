/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { MapSectionBodyMain } from '../MapSectionBodyMain';

describe('MapSectionBodyMain', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(<MapSectionBodyMain/>).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
