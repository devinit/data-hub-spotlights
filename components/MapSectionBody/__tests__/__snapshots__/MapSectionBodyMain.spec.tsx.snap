// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapSectionBodyMain renders correctly 1`] = `
<div
  className="spotlight__main spotlight__interactive"
/>
`;
